tests/api2/test_011_user.py

```diff
@@ -212,7 +212,7 @@ def test_002_verify_user_exists_in_pwd(request):
     assert pw['pw_dir'] == VAR_EMPTY
 
     # At this point, we're not an SMB user
```

Comment:
We expect `sid` to be `None` because we created the user with `smb` = `False`, right?

Perhaps update `test_005_convert_non_smbuser_to_smbuser` to rerun this `user.get_user_obj` and validate the `sid` is **not** `None`?